lib/frontend_helpers/utils.rb

```diff
@@ -54,7 +54,7 @@ def active_path?(path, match_option)
       when :partial
         request.path.include?(path)
       when String
-        request.path.include?(match_option)
+        request.path.include?(match_option) && request.path.include?(path)
```

Comment:
Esta solucion es para resolver el caso del sidebar bar, donde se tienen multiples paths presentes pero solo uno se relaciona al path del request. 

E.g., 

`<%= render SidebarMenu::Component.new(class: "sidebar-container") do |c| %>
    <%= c.item(title: t('.sidebar.home'), icon: 'dashboard', href: user_home_index_path, match: params[:format]) %>
    <%= c.item(title: t('.sidebar.orders'), icon: 'cutlery-alt', href: user_orders_path, match: params[:format]) %>
<% end %>`

En ese caso se tendrian dos paths diferentes llamando a active link, por lo que para diferencialos agregue el AND.

Reply:
esto no lo necesitas, si te fijas hay una opcion en la linea 54 que dice: `when :partial` y contiene la misma exacta linea de codigo que acabas de agregar